### ISSUE TITLE
No alt text for move rule arrows

### ISSUE THREAD

**Issue by Lorrainbow** [microbit-apps/microcode-v2#28]
Version: 2.5.47
Issue type: accessibility
Hardware: elecfreaks (smaller screen)

**Replicate**
1. From the home screen select the save icon
2. Select any of the saved files (1, 2 or 3). I chose 1
3. From the blank screen create two rules. 
4. Select the select box next to the first rule

**Result:** The select box has the icons "add rule", "delete rule" and a down arrow. The down arrow has no alt text
**Expected:** The down arrow should have alt text to indicate what it does, e.g. "move rule"

**Comment by thomasjball** [comment]
fixed

**Comment by Lorrainbow** [comment]
fixed
